app/controllers/instances_controller.rb

```diff
@@ -8,6 +8,10 @@ def create # Create the game instance
         status: "pending"
       )
 
+      # Simplistic pin number
+      @instance.pin = 100_000 + @instance.id
+      @instance.save
```

Comment:
I think this is good and simple for now. Later on maybe we can create our own pin mechanism or use something like this: https://github.com/spannerinworks/easy-pin/.